src/tsoder_app.erl

```diff
@@ -14,7 +14,19 @@
 %% API
 %%====================================================================
 
+log_file_name(Suffix) ->
+    string:join(["log_", Suffix, ".txt"], "").
```

Comment:
@ForNeVeR thanks for you comment! Unfortunately, `lists:join` in the current version of Erlang we support (18). I'll leave a comment to replace it with `lists:join` when Erlang version is > 18.